thirdparty/pip_tools-5.3.0-py2.py3-none-any.whl.ABOUT

```diff
@@ -0,0 +1,16 @@
+about_resource: pip_tools-5.3.0-py2.py3-none-any.whl 
```

Comment:
We do not want pip-tools to be included in the thirdparty/ instead you should use a requirements.in file dedicated to it and the other dev tools and that file should be compiled to a requirements.txt pinned with hashes used only for dev

Reply:
that must be there for generating hashes. After that you can remove whole thirdparty, i don't care.